src/main/java/Azir/Task.java

```diff
@@ -0,0 +1,42 @@
+package Azir;
+
+/**
+ * Representation of a task
+ * Able to set if a task has been completed
+ */
+public class Task {
```

Comment:
Would you be open to considering making Task an abstract class, since you are likely to not have to instantiate it?

Reply:
Sure. That is a great idea.